mmv1/third_party/terraform/services/logging/resource_logging_bucket_config.go

```diff
@@ -36,6 +36,14 @@ var loggingBucketConfigSchema = map[string]*schema.Schema{
 		Computed:    true,
 		Description: `An optional description for this bucket.`,
 	},
+	"restricted_fields": {
```

Comment:
We need a test for this resource as well right? not just the project version

Reply:
True

Reply:
This should be resolved now